### PR TITLE
Create say_hi.py

### PR DESCRIPTION
**Pull request by oumatou**
just created a file to say hi to strangers

### CODE REVIEW

**Comment by oumatou** [comment]
can you add it to ur repo

File: say_hi.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print('hello stranger')
```
Comment (oumatoulacen) [inline say_hi.py:1]: can you add hello everyone instaed

Comment (oumatou) [reply say_hi.py]: i did

Comment (oumatoulacen) [reply say_hi.py]: good job

**Pull request merged**